doc/source/whatsnew/v0.21.0.txt

```diff
@@ -778,6 +778,7 @@ I/O
 - Bug in :func:`read_html` where import check fails when run in multiple threads (:issue:`16928`)
 - Bug in :func:`read_csv` where automatic delimiter detection caused a ``TypeError`` to be thrown when a bad line was encountered rather than the correct error message (:issue:`13374`)
 - Bug in ``DataFrame.to_html()`` with ``notebook=True`` where DataFrames with named indices or non-MultiIndex indices had undesired horizontal or vertical alignment for column or row labels, respectively (:issue:`16792`)
+- Bug in ``DataFrame.to_html()`` in which there was no validation of the ``justify`` parameter (:issue:`17527`)
```

Comment:
Would it be preferable to use <code>:meth:\`DataFrame.to_html\`</code> to provide a link?  Possibly on the line above too?

Reply:
Absolutely!  Thanks for pointing that out.